frontend/src/reducers/user.js

```diff
@@ -0,0 +1,82 @@
+import { batch } from 'react-redux';
+import { createSlice } from '@reduxjs/toolkit';
+import { API_URL } from 'reusable/urls';
+
+export const user = createSlice({
+  name: 'user',
+  initialState: {
+    email: null,
+    accessToken: null,
+    errors: null,
+  },
+  reducers: {
+    setEmail: (store, action) => {
+      store.email = action.payload;
+    },
+    setAccessToken: (store, action) => {
+      store.accessToken = action.payload;
+    },
+    setErrors: (store, action) => {
+      store.errors = action.payload;
+    },
+  },
+});
```

Comment:
You need to store your data in the local storage to avoid logging out your user after refreshing the page.